otx/algorithms/detection/adapters/mmdet/utils/exporter.py

```diff
@@ -1,39 +1,35 @@
-"""Export task for OTX Detection with MMDET."""
+"""Exporter for OTX Detection task with MMDETECTION training backend."""
 # Copyright (C) 2023 Intel Corporation
 # SPDX-License-Identifier: Apache-2.0
 #
 
 import numpy as np
-from mmcv.runner import wrap_fp16_model
 
-from otx.algorithms.common.adapters.mmcv.tasks.exporter_mixin import ExporterMixin
-from otx.algorithms.common.adapters.mmcv.tasks.registry import STAGES
-from otx.algorithms.common.adapters.mmdeploy.utils import sync_batchnorm_2_batchnorm
+from otx.algorithms.common.adapters.mmcv.tasks.exporter import Exporter
+from otx.algorithms.common.adapters.mmdeploy.utils.utils import (
+    sync_batchnorm_2_batchnorm,
+)
 from otx.algorithms.common.utils.logger import get_logger
-
-from .stage import DetectionStage
+from otx.algorithms.detection.adapters.mmdet.utils.builder import build_detector
 
 logger = get_logger()
 
 
-@STAGES.register_module()
-class DetectionExporter(ExporterMixin, DetectionStage):
-    """Export class for object detection."""
+class DetectionExporter(Exporter):
```

Comment:
What is the differences between `MMDetectionTask.export()`?
As far as I remember, did we decide to delete this ?

Reply:
I want to bring this codes to export, but it's hard to bring previous exporter_mixin code. And I think exporter is already well implemented, so it is not necessary to bring to task.

Reply:
> I want to bring this codes to export, but it's hard to bring previous exporter_mixin code. And I think exporter is already well implemented, so it is not necessary to bring to task.

I agree with this point of view, however, coexisting of two similar function could make users to confuse. (Detectiontask.export(), `DetectionExporter`).

Then, do you agree with the suggestion that removes `DetectionExporter` in the future?

Reply:
I hope, but I'm not sure

Reply:
And also, there are a lot of works for MMDetectionTask.export() except works for DetectionExporter

Reply:
It looks like it needs to be refactored with mixin in the future. But it seems hard to do in this PR.

Reply:
I agree, I think the whole Exporter part should be moved to _export_model function(currently not existed).